ledger-tool/src/main.rs

```diff
@@ -1930,7 +1930,16 @@ fn main() {
                 Arg::with_name("skip_poh_verify")
                     .long("skip-poh-verify")
                     .takes_value(false)
-                    .help("Skip ledger PoH verification"),
+                    .help(
+                        "Skip ledger PoH and transaction verification.  Use --skip-verification \
```

Comment:
Looks like we don't have a convention for deprecated cli args, but maybe it is better to start with "Deprecated, please use ..."

Reply:
Done.